paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSessionCustomerSheetActivityTest.kt

```diff
@@ -396,6 +404,9 @@ class CustomerSessionCustomerSheetActivityTest {
             host("api.stripe.com"),
             method("GET"),
             path("/v1/payment_methods"),
+            query("type", "card"),
+            query("customer", "cus_12345"),
+            query("limit", "100")
```

Comment:
Do we care about this here?

Reply:
This came up a while back when our limit of `10` was too small since there's an off-chance customers have more. Figured we should be aware of changes to the limit here as well since it can affect duplicates.